docs/hardware_keyboard_guidelines.md

```diff
@@ -88,12 +88,14 @@ The `config.h` files can also be placed in sub-folders, and the order in which t
       * `keyboards/top_folder/sub_1/sub_2/sub_3/config.h`
         * `keyboards/top_folder/sub_1/sub_2/sub_3/sub_4/config.h`
           * `users/a_user_folder/config.h`
-          * `keyboards/top_folder/keymaps/a_keymap/config.h`
-        * `keyboards/top_folder/sub_1/sub_2/sub_3/sub_4/post_config.h`
-      * `keyboards/top_folder/sub_1/sub_2/sub_3/post_config.h`
-    * `keyboards/top_folder/sub_1/sub_2/post_config.h`
-  * `keyboards/top_folder/sub_1/post_config.h`
-* `keyboards/top_folder/post_config.h`
+            * `keyboards/top_folder/keymaps/a_keymap/config.h`
```

Comment:
In the following description, the nesting levels of SUB_X/config.h and SUB_X/post_config.h do not match.

* `keyboards/top_folder/config.h`
  * `keyboards/top_folder/sub_1/config.h`
    * `keyboards/top_folder/sub_1/sub_2/config.h`
      * `keyboards/top_folder/sub_1/sub_2/sub_3/config.h`
        * `keyboards/top_folder/sub_1/sub_2/sub_3/sub_4/config.h`
          * `users/a_user_folder/config.h`
            * `keyboards/top_folder/keymaps/a_keymap/config.h`
          * `keyboards/top_folder/sub_1/sub_2/sub_3/sub_4/post_config.h`
        * `keyboards/top_folder/sub_1/sub_2/sub_3/post_config.h`
      * `keyboards/top_folder/sub_1/sub_2/post_config.h`
    * `keyboards/top_folder/sub_1/post_config.h`
  * `keyboards/top_folder/post_config.h`
* `users/a_user_folder/post_config.h`

I feel it would be better to match the nest levels as follows.

* `keyboards/top_folder/config.h`
  * `keyboards/top_folder/sub_1/config.h`
    * `keyboards/top_folder/sub_1/sub_2/config.h`
      * `keyboards/top_folder/sub_1/sub_2/sub_3/config.h`
        * `keyboards/top_folder/sub_1/sub_2/sub_3/sub_4/config.h`
          * `users/a_user_folder/config.h`
          * `keyboards/top_folder/keymaps/a_keymap/config.h`
        * `keyboards/top_folder/sub_1/sub_2/sub_3/sub_4/post_config.h`
      * `keyboards/top_folder/sub_1/sub_2/sub_3/post_config.h`
    * `keyboards/top_folder/sub_1/sub_2/post_config.h`
  * `keyboards/top_folder/sub_1/post_config.h`
* `keyboards/top_folder/post_config.h`
* `users/a_user_folder/post_config.h`